src/bid_order.rs

```diff
@@ -227,23 +227,22 @@ pub fn migrate_bid_orders(
 
     require_version(">=0.16.2", &current_version)?;
 
-    // Migrate BidOrderV2 -> BidOrderV2
-    if VersionReq::parse(">=0.16.2, <0.18.2")?.matches(&current_version) {
-        // get all bid ids
-        let existing_bid_order_ids: Vec<Vec<u8>> = get_bid_storage_read::<BidOrderV2>(store)
+    // Migrate only BidOrderV2 -> BidOrderV3
+    if VersionReq::parse(">=0.16.2, <0.19.1")?.matches(&current_version) {
```

Comment:
Fixed version range check

Reply:
silly question, why we need < version check? do we need remember to bump the number all the time?